<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>237</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="hexTab">
      <attribute name="title">
       <string>Hex pattern</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="hexLabel">
         <property name="text">
          <string>Hex pattern:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="hexEdit">
         <property name="inputMask">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="hexGroupBox">
         <property name="title">
          <string>Search scope</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="hexScopeFullRadioButton">
            <property name="text">
             <string>Header and body</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="hexScopeHeaderRadioButton">
            <property name="text">
             <string>Header only</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="hexScopeBodyRadioButton">
            <property name="text">
             <string>Body only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="guidTab">
      <attribute name="title">
       <string>GUID</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="guidLabel">
         <property name="text">
          <string>GUID:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="GuidLineEdit" name="guidEdit">
         <property name="inputMask">
          <string>xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</string>
         </property>
         <property name="text">
          <string>........-....-....-....-............</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="guidGroupBox">
         <property name="title">
          <string>Search scope</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="guidScopeFullRadioButton">
            <property name="text">
             <string>Header and body</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="guidScopeHeaderRadioButton">
            <property name="text">
             <string>Header only</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="guidScopeBodyRadioButton">
            <property name="text">
             <string>Body only</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="textTab">
      <attribute name="title">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="textLabel">
         <property name="text">
          <string>Text:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="textEdit"/>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="textGroupBox">
         <property name="title">
          <string>Text search options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QCheckBox" name="textUnicodeCheckBox">
            <property name="text">
             <string>Unicode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="textCaseSensitiveCheckBox">
            <property name="text">
             <string>Case sensitive</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GuidLineEdit</class>
   <extends>QLineEdit</extends>
   <header>guidlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>hexEdit</tabstop>
  <tabstop>hexScopeFullRadioButton</tabstop>
  <tabstop>hexScopeHeaderRadioButton</tabstop>
  <tabstop>hexScopeBodyRadioButton</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>textUnicodeCheckBox</tabstop>
  <tabstop>textCaseSensitiveCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SearchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SearchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
